pkg/controller/hostpathprovisioner/scc.go

```diff
@@ -45,7 +45,8 @@ func (r *ReconcileHostPathProvisioner) reconcileSecurityContextConstraints(reqLo
 		}
 	} else {
 		if err := r.deleteSCC(MultiPurposeHostPathProvisionerName); err != nil {
-			return reconcile.Result{}, err
+			reqLogger.Error(err, "unable to delete SCC")
```

Comment:
Why do we not error out anymore when the delete fails? maybe just IgnoreNotFound?